pkg/convert/pot.go

```diff
@@ -0,0 +1,144 @@
+package convert
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"strings"
+
+	"go.expect.digital/translate/pkg/model"
+	"go.expect.digital/translate/pkg/pot"
+)
+
+type PoTag string
```

Comment:
We expose only `func From<std>` and `func To<std>`.  Does `pot` package have `msgid` and `msgstr`?